roboflow/core/project.py

```diff
@@ -546,6 +551,21 @@ def _annotation_params(self, annotation_path):
             )
         return annotation_name, annotation_string
 
+    def _parse_upload_error(self, error: rfapi.UploadError) -> str:
+        dict_part = str(error).split(": ", 2)[2]
+        if re.search(r"'\w+':", dict_part):
+            dict_part = dict_part.replace("True", "true")
+            dict_part = dict_part.replace("False", "false")
+            dict_part = dict_part.replace("None", "null")
```

Comment:
What if `re.search(r"'\w+':", dict_part)` is false?

We still wish to load from json, but since we didn't run `dict_part = dict_part.replace("True", "true")`, it might not be valid.

Reply:
I guess now I got it right